.circleci/config.yml

```diff
@@ -369,48 +369,20 @@ jobs:
       - setup_remote_docker:
           docker_layer_caching: true
       - run: |
-          GAIAD_VERSION=''
-          if [ "${CIRCLE_BRANCH}" = "master" ]; then
-            GAIAD_VERSION="stable"
-          elif [ "${CIRCLE_BRANCH}" = "develop" ]; then
-            GAIAD_VERSION="develop"
-          fi
-          if [ -z "${GAIAD_VERSION}" ]; then
-            docker build .
-          else
-            docker build -t tendermint/gaia:$GAIAD_VERSION .
-            docker login --password-stdin -u $DOCKER_USER <<<$DOCKER_PASS
-            docker push tendermint/gaia:$GAIAD_VERSION
-          fi
-
-  docker_tagged:
```

Comment:
Aren't we going to loose the tagged images? One docker image at tag creation:
CIRCLE_TAG correspond to the git tag and the event is triggered only when the tag is created, where CIRCLE_BRANCH will always refresh this image if we push on that branch

Reply:
We moved to a release branch model instead of tags. Images will be created when the release branch is created.

Reply:
But the tags are still being created right?

Reply:
According to our [release procedure](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#release-procedure) they are not being created anymore.

Reply:
But then I see that the point releases do have tags? I guess this should be discussed in the SDK standup tomorrow to have clarity.

Reply:
Tags should be created for all releases (major and point releases).

Reply:
Will bring this up today 👍